src/value.h

```diff
@@ -154,10 +154,15 @@ static inline RSValue *RSValue_IncrRef(RSValue *v) {
   return v;
 }
 
-#define RSValue_Decref(v)                                         \
-  if (!__atomic_sub_fetch(&(v)->refcount, 1, __ATOMIC_RELAXED)) { \
-    RSValue_Free(v);                                              \
+static inline void RSValue_Decref(RSValue* v) {
+  if (__atomic_sub_fetch(&(v)->refcount, 1, __ATOMIC_RELAXED) == 0) {
+    RSValue_Free(v);
   }
+}
+
+void RSValue_IncrRef_Rust(RSValue* v);
```

Comment:
I would prefer maybe not to name them Rust, but call the static one with some _ underscore to imply "private"?